src/Core/Persistence/JsonFileHandler.cs

```diff
@@ -27,18 +27,41 @@ namespace SonarLint.VisualStudio.Core.Persistence;
 
 public interface IJsonFileHandler
 {
+    /// <summary>
+    /// Tries to read the json file and to deserialize the model.
+    /// </summary>
+    /// <typeparam name="T">The type of the model that will be serialized</typeparam>
+    /// <param name="filePath">The path to the file</param>
+    /// <param name="content">The content of the file deserialized to the provided type</param>
+    /// <returns>True if the file could be read and the model could be serialized successfully. False otherwise</returns>
     bool TryReadFile<T>(string filePath, out T content) where T : class;
```

Comment:
Is this method going to be used anywhere?

Reply:
For now, I think not. I could remove it.